### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Titech0**

### CODE REVIEW

**Comment by YegorKorenkov** [comment]
Please attach a link to the demo and tests. Don't forget about it

File: src/index.html [hunk 1]
```diff
@@ -2,16 +2,267 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
     <script type="text/javascript" src="./main.js"></script>
+    <header class="header"></header>
+
+    <main class="main">
+      <div class="content">
```
Comment (YegorKorenkov) [inline src/index.html:15]: A lot of redundant classes and element

File: src/index.html [hunk 2]
```diff
@@ -2,16 +2,267 @@
...
+        name="taskForm"
+        >
+          <fieldset class="group personalInfo">
+            <legend class="legend personalInfo__legend">
+              Personal information
+            </legend>
+            <div class="content personalInfo__content">
+
+              <section class="section surname__section">
+                <div class="section__content">
```
Comment (YegorKorenkov) [inline src/index.html:27]: Why you need this div, is it dont do nothing?

File: src/index.html [hunk 3]
```diff
@@ -2,16 +2,267 @@
...
+      <div class="content">
+        <form
+        action="http://localhost:8080/api" method="post"
+        name="taskForm"
+        >
+          <fieldset class="group personalInfo">
+            <legend class="legend personalInfo__legend">
+              Personal information
+            </legend>
+            <div class="content personalInfo__content">
```
Comment (YegorKorenkov) [inline src/index.html:24]: Delete this div. You added a lot of redundant classes, but they don do nithing. Please keep only two classes: 
1. for fieldset with margin between groups
2. for form-field with margin between inputs

File: src/index.html [hunk 4]
```diff
@@ -2,16 +2,267 @@
...
+                </div>
+              </section>
+
+              <section class="section isRecommendUs__section">
+                <div class="section__content">
+                  <label for="recommendUs">
+                    Would you recommend us?:
+                  </label>
+                  <select name="recommendUs">
+                    <option value="true">Yes</option>
```
Comment (YegorKorenkov) [inline src/index.html:244]: You should have only yes and no options

File: src/index.html [hunk 5]
```diff
@@ -2,16 +2,267 @@
...
+                    <option value="true">Yes</option>
+                  </select>
+                </div>
+              </section>
+            </div>
+          </fieldset>
+          <div class="button__wrapper">
+            <button
+              class="submitButton"
```
Comment (YegorKorenkov) [inline src/index.html:255]: Redundant class & id

File: src/index.html [hunk 6]
```diff
@@ -2,16 +2,267 @@
...
+                    id="isTimeGoToBed"
+                  >
+                </div>
+              </section>
+              <section class="section isFavBrandOfCars__section">
+                <div class="section__content">
+                  <label for="isFavBrandOfCars">
+                    What are your favorite brand of cars?
+                  </label>
+                  <select name="isFavBrandOfCars" multiple>
```
Comment (YegorKorenkov) [inline src/index.html:196]: Provide attr id for correct work label's attr for

File: src/index.html [hunk 7]
```diff
@@ -2,16 +2,267 @@
...
+
+          <fieldset class="group aboutYou">
+            <legend class="legend aboutYou__legend">
+              An interesting fact about you?
+            </legend>
+            <div class="content aboutYou__content">
+
+              <section class="section isLoveCat__section">
+                <div class="section__content">
+                  <label for="isLoveCat">
```
Comment (YegorKorenkov) [inline src/index.html:144]: Redundant label

File: src/index.html [hunk 8]
```diff
@@ -2,16 +2,267 @@
...
+                    Do you love cats?
+                    <input
+                      type="radio"
+                      name="isLoveCat"
+                      id="isLovecat--true"
```
Comment (YegorKorenkov) [inline src/index.html:149]: Please provide id in camelCase fromat fro example lova/notLove

**Review by YegorKorenkov — COMMENTED** [review]
Please read comments. Delete all unused classe, redundant elements. Add class to element if you want add some css styles. It's just that you don't have to do it. And please add links to demo/tests.

**Review by etojeDenys — COMMENTED** [review]
please, add DEMO & REPORT link for your PR

or feel free to ask about it in the slack chat

**Pull request closed without merging**